src/components/modals/login-modal/login-content.less

```diff
@@ -0,0 +1,106 @@
+.ant-btn.ant-btn-primary.login-button {
+  position: absolute;
+  left: 40.8%;
+  right: 40.8%;
+  top: 58.99%;
+  bottom: 32.58%;
+  background: #505050;
+  border-radius: 6px;
+  height: 45px;
+  width: 115px;
+}
+
+.forgot-password-text {
+  height: 44px;
+  width: 480px;
+  position: absolute;
+  left: 11.68%;
+  right: 11.52%;
+  top: 72.28%;
+  bottom: 19.48%;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 156%;
+  white-space: nowrap;
+  text-align: center;
+  color: #000000;
+}
+
+.page-link {
+  height: 14px;
+  color: black;
+  white-space: nowrap;
+  background-color: transparent;
+  border: transparent;
+  font-family: Roboto;
```

Comment:
We dont want to set font family! The font family should not be Roboto, we changed to Raleway on the design! Our typography does this automatically so we can just remove this.